staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -20,6 +20,9 @@ import (
 	"fmt"
 
 	"github.com/spf13/pflag"
+	"k8s.io/apiserver/pkg/server/healthz"
```

Comment:
nit: want an empty line before these. we normally put a gap between dependencies and in-project deps